packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -335,16 +336,10 @@ export default class PciInstancesAddController {
     };
   }
 
-  fetch3AZAvailability() {
-    return this.PciProjectsProjectInstanceService.getProductAvailability(
+  async fetchRegionTypeAvailability() {
```

Comment:
Same here, as there is no filter anymore, should be plural:
```suggestion
  async fetchRegionsTypeAvailability() {
```